### PR TITLE
v1.0.0: Add RCpair copy! for complex. Rename `region`->`dims`. Remove deprecation.

### PR DESCRIPTION
**Pull request by IanButterworth**
Part of https://github.com/JuliaImages/ImageFiltering.jl/pull/271
- add RCpair `copy!` for complex required for irfft
- rename `region` arg to `dims` as per https://github.com/JuliaImages/ImageFiltering.jl/pull/271#discussion_r1509140217
- remove deprecated functions as per https://github.com/JuliaImages/ImageFiltering.jl/pull/271#discussion_r1509141437

### CODE REVIEW

**Comment by IanButterworth** [comment]
Or maybe it doesn't need to move org?

**Comment by IanButterworth** [comment]
Bump

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -13,10 +13,13 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.1'
-#          - 'nightly'
+          - '1.6'
+          - '1'
+          - 'nightly'
```
Comment (timholy) [inline .github/workflows/CI.yml:18]: I don't like testing against `nightly`, see https://discourse.julialang.org/t/why-do-packages-run-continuous-integration-tests-on-julia-nightly/101208/3?u=tim.holy. I know this isn't in General but it probably will be soon.

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -13,10 +13,13 @@ jobs:
...
         os:
           - ubuntu-latest
+          - windows-latest
+          - macOS-latest
```
Comment (timholy) [inline .github/workflows/CI.yml:22]: Yeah, it makes sense to do this for a package like this one.

File: Project.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name = "RFFT"
 uuid = "3bd9afcd-55df-531a-9b34-dc642dce7b95"
 authors = ["Tim Holy <tim.holy@gmail.com>"]
-version = "0.1.0"
+version = "0.2.0"
```
Comment (timholy) [inline Project.toml:4]: Unless you expect more breaking changes soon, I suspect we should just go for 1.0.

**Review by timholy — APPROVED** [review]
Minor tweaks, then merge at will.

**Comment by timholy** [comment]
I agree with the plan. My move of IncrementalSVD revealed that we'll have to change the UUID, as General has a guard against registering a package with the same uuid as in HolyLabRegistry.

I will also enhance the README before we release in General, as it seems every so slightly threadbare now 😆

File: .github/workflows/CI.yml (hunk 1 above)
Comment (IanButterworth) [inline .github/workflows/CI.yml:18]:
```suggestion
```

File: Project.toml (hunk 3 above)
Comment (IanButterworth) [inline Project.toml:4]:
```suggestion
version = "1.0.0"
```

**Comment by IanButterworth** [comment]
github actions runners seem to be having codecov upload issues but tests pass

**Pull request merged**